docs/ignore.md

```diff
@@ -24,7 +24,7 @@ coverage
 
 It’s recommended to have a `.prettierignore` in your project! This way you can run `prettier --write .` to make sure that everything is formatted (without mangling files you don’t want, or choking on generated files). And – your editor will know which files _not_ to format!
 
-By default prettier ignores files in version control systems directories (".git", ".svn" and ".hg") and `node_modules` (if [`--with-node-modules` CLI option](cli.md#--with-node-modules) not specified)
+By default prettier ignores files in version control systems directories (".git", ".svn" and ".hg") and `node_modules` (if [`--with-node-modules` CLI option](cli.md#--with-node-modules) not specified). Prettier will also follow rules specified in .gitignore if it exists in the same directory from which it is run.
```

Comment:
We also ignores `.sl`  https://github.com/prettier/prettier/blob/56945ecf12a5b2e3b70e13bbcec956ccaf7bd460/src/cli/expand-patterns.js#L47

Reply:
If we are going to mention `.gitignore` here, maybe we should put `.prettierignore` here too.